ducktape/tests/test.py

```diff
@@ -52,12 +54,22 @@ def min_cluster_size(self):
         """
         return self.test_context.services.num_nodes()
 
-    def setUp(self):
+    def setup(self):
```

Comment:
It's actually originally inherited from the unittest module: https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUp which afaik violates PEP styling in favor of matching the extremely widespread styling of jUnit and most xUnit variants.